README.md

```diff
@@ -200,17 +212,20 @@ services:
     - /opt/docker/subspace:/data
    restart: always
    environment:
-    - SUBSPACE_HTTP_HOST=subspace.example.org
-    - SUBSPACE_LETSENCRYPT=true
-    - SUBSPACE_HTTP_INSECURE=false
-    - SUBSPACE_HTTP_ADDR=":80"
-    - SUBSPACE_NAMESERVER=1.1.1.1
-    - SUBSPACE_LISTENPORT=51820
-    - SUBSPACE_IPV4_POOL=10.99.97.0/24
-    - SUBSPACE_IPV6_POOL=fd00::10:97:0/64
-    - SUBSPACE_IPV4_GW=10.99.97.1
-    - SUBSPACE_IPV6_GW=fd00::10:97:1
-    - SUBSPACE_IPV6_NAT_ENABLED=1
+    - 'SUBSPACE_HTTP_HOST=subspace.example.org'
+    - 'SUBSPACE_BACKLINK=/'
+    - 'SUBSPACE_LETSENCRYPT=true'
+    - 'SUBSPACE_HTTP_INSECURE=false'
+    - 'SUBSPACE_HTTP_ADDR=:80'
+    - 'SUBSPACE_NAMESERVER=1.1.1.1'
+    - 'SUBSPACE_LISTENPORT=51820'
+    - 'SUBSPACE_ENDPOINT_HOST=you-can-use-another-hostname.subspace.example.com'
+    - 'SUBSPACE_NETWORK_IPV4=10.99.97.0/24'
+    - 'SUBSPACE_NETWORK_IPV6=fd00::10:97:0/64'
+    - 'SUBSPACE_IPV6_NAT_ENABLED=1'
+    - 'SUBSPACE_ENABLE_DNSMASQ=1'
+    - 'SUBSPACE_ALLOWED_IPS=0.0.0.0/0, ::/0'
+    - 'SUBSPACE_THEME=green'
```

Comment:
can you remove the `'` as they aren't required in docker-compose and its confusing the diff

Reply:
It is because if we wrote like:

```yaml
    - SUBSPACE_ALLOWED_IPS="0.0.0.0/0, ::/0"
```

I found that `allowedIPs` in golang will be `"0.0.0.0/0, ::/0"` (incldues quotes!), and it makes a syntax errors in generated config files.
I don't know this is a bug in docker-compose or expected behavior.

Could you tell me your idea about it?